src/components/templates/Tooltip/Tooltip.tsx

```diff
@@ -8,11 +8,13 @@ export const Tooltip = ({
     children,
     align = 'center',
     maxWidth = 'large',
+    severity,
```

Comment:
With the class name suggestion below (line 61), we can give a default `severity = 'info'` here so the styling is applied